destination/format/csv.go

```diff
@@ -489,3 +478,24 @@ func isDateOrTimeType(in string) bool {
 		return false
 	}
 }
+
+func mapAvroToSnowflake(field *avro.Field) string {
+	p := field.Type().(*avro.PrimitiveSchema)
+	switch p.Logical().Type() {
+	case avro.Decimal:
+		return SnowflakeFloat
+	case avro.UUID:
+		return SnowflakeVarchar
+	case avro.Date:
+		return SnowflakeDate
+	case avro.TimeMillis:
```

Comment:
Pg also sends `LocalTimestampMicros` which is I think SnowflakeTimestampNTZ (I think this is the local time variety in snowflake?)

Reply:
that is covered by `avro.TimeMicros` right? unless you mean we need to pass down the timezone for timezone-specific scenarios

Reply:
I think it maps to different types. The local timestamp in avro does not carry a TZ component.  Timestamp with TZ and timestamp without TZ